source/dialog/fob/fast_travel.sqf

```diff
@@ -27,8 +27,11 @@ _selected_fob = _list select _index;
 _selected_fob_pos = _selected_fob select 0;
 _selected_fob_name = _selected_fob select 1;
 
-player setpos _selected_fob_pos;
+//Finding a safe place to spawn
+_selected_fob_pos_safe = [_selected_fob_pos, 0, 25, 2, 0, 20, 0] call BIS_fnc_findSafePos;
```

Comment:
change looks good to me, merge it.
soo the FOB position is the one where the FOB is created or where the FOB is requested?

**IF** you want to double check that , you could add a marker to where the FOB is placed (can't remember if we do that now) and a marker to the position of the player when he/she requests the FOB (make sure to add it inside a if (debug) condition)

Reply:
It looks like from support\fob.sqf that the var that's passed into the array is the result of a BIS_fnc_findSafePos and i can see that's what's passed into the marker on the map too. (_foundPickupPos in fob.sqf)

My testing supports this as players always spawn within the set no. of meters of the FOB itself, and not the area the request originated from.